Services/DataX.Flow/DataX.Flow.SchemaInference.Tests/SchemaUtilityTests.cs

```diff
@@ -0,0 +1,98 @@
+﻿// *********************************************************************
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License
+// *********************************************************************
+using DataX.Utilities.Blob;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace DataX.Flow.SchemaInference.Tests
+{
+    [TestClass]
+    public class SchemaUtilityTests
+    {
+        [TestMethod]
+        public void BlobPathPrefixTest()
+        {
+            List<Tuple<string, string, string, string>> blobPaths = new List<Tuple<string, string, string, string>>
+            {
+                new Tuple<string, string, string, string>(
+                    @"wasbs://mycontainer@mysa.blob.core.windows.net/mypath/mypath2/mypath3/{yyyy}/{MM}/{dd}",
```

Comment:
Does this pattern still work with the regex updates?

Reply:
No regex update was needed. The current logic just works with this {yyyy/MM/dd}. Add a couple tests to test {yyyy/MM/dd} and {yyyy/MM/dd}/test.